src/ImageSharp/Formats/Png/Filters/PaethFilter.cs

```diff
@@ -106,7 +128,7 @@ public static void Encode(Span<byte> scanline, Span<byte> previousScanline, Span
         /// The <see cref="byte"/>.
         /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private static byte PaethPredicator(byte left, byte above, byte upperLeft)
+        private static byte PaethPredictor(byte left, byte above, byte upperLeft)
```

Comment:
It's funny... Whatever source I used when first writing these filters had "predicator", it's definitely "predictor" though, they use that word in the spec! 😄